frontend/src/components/profile/TrainingTips.tsx

```diff
@@ -0,0 +1,169 @@
+                                if you’re not regularly playing tournaments.
+                                Over-the-board play is best for improvement, but you can
+                                also play classical games online!
+                            </Box>
+
+                            <Box>
+                                Join one of Dojo's current tournaments to get some serious
+                                games in.
+                            </Box>
+                        </TrainingTipsCard>
+                        <TrainingTipsCard
+                            name='Annotate your Games'
+                            onClick={() => goto('/games/import')}
+                            icon={icons.Annotations}
+                        >
+                            If you’ve played some classical games recently, it’s time to
+                            analyze those games! Use our Game Editor to start annotating
+                            and remember to publish your analysis when you’re finished!
+                        </TrainingTipsCard>
+                        <TrainingTipsCard
+                            name='Tactics Test'
+                            onClick={() => goto('/tests')}
+                            icon={icons.Tactics}
+                        >
+                            After Games & Analysis, Tactics is the next most important
+                            category. Start with a Dojo Tactics Test, or work on
+                            completing one of the tasks in the Tactics section of the
+                            Training Plan below (Polgar Mates, Puzzle Rush, etc.)
+                        </TrainingTipsCard>
+                        <TrainingTipsCard
+                            name='Suggested Tasks'
+                            onClick={() => closeDialog()}
+                            icon={icons['Suggested Tasks']}
+                        >
+                            Work through one of the suggested tasks listed here. Make sure
+                            to mark your progress and log your hours!
+                        </TrainingTipsCard>
+                    </Stack>
+                </Stack>
+            </DialogContent>
+            <DialogActions>
+                <Button onClick={onClose}>Close</Button>
+            </DialogActions>
+        </Dialog>
+    );
+}
+interface TrainingTipsCardProps {
+    name: string;
+    children: ReactNode;
+    icon:
+        | ((props: SvgIconProps) => JSX.Element)
+        | (OverridableComponent<SvgIconTypeMap> & { muiName: string });
+    onClick: () => void;
+}
+
+const TrainingTipsCard = ({ name, children, icon, onClick }: TrainingTipsCardProps) => {
+    const Icon = icon;
+
+    return (
+        <Grid2
+            size={{
+                xs: 12,
+                sm: 6,
+            }}
+        >
+            <Card sx={{ height: 1 }}>
+                <CardActionArea sx={{ height: 1 }} onClick={onClick}>
```

Comment:
Use `href` instead of `onClick` and add the `component` parameter. Pass it `Link` from `@/components/navigation/Link`. That should fix the engine load error.